Server/JokesOnYou.Web.Api/Exceptions/AppException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace JokesOnYou.Web.Api.Exceptions
+{
+    public class AppException : Exception
+    {
+        public AppException() : base()
```

Comment:
don't think the base here is needed ?
https://docs.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-user-defined-exceptions

Reply:
Correct. Its a matter of inheritance and usage of base keyword tho.